modules/Activities/tpls/PopupBody.tpl

```diff
@@ -0,0 +1,286 @@
+                                    <td>{$activity.description}
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <!--  END: description -->
+
+
+                {/foreach}
+
+                </tbody>
+                <!-- END: row -->
+            </table>
+
+        </div>
+
+        <div role="tabpanel" class="tab-pane" id="calls">
+            <table class="list view table-responsive subpanel-table">
+                <thead>
+                <tr class="footable-header">
+                    <th><img src="include/images/blank.gif" width="1" height="1" alt=""></th>
+                    <th>{$mod.LBL_LIST_SUBJECT}</th>
+                    <th>{$mod.LBL_LIST_STATUS}</th>
+                    <th>{$mod.LBL_LIST_CONTACT}</th>
+                    <th>{$mod.LBL_LIST_DATE}</th>
+                </tr>
+                </thead>
+                <tbody>
+                {foreach from=$callsList key=k item=activity}
+
+                    <!-- BEGIN: row -->
+                    <tr>
+                        <td><img src={$activity.image}></td>
+                        <td>{$activity.name} {$activity.attachment}</td>
+                        <td>{$activity.type} {$activity.status}</td>
+                        <td>{$activity.contact_name}</td>
+                        <td>{$activity.date_type} {$activity.date_modified}</td>
+                    </tr>
+                    <!--  BEGIN: description -->
+                    <tr>
+                        <td></td>
+                        <td>
+                            <table>
+                                <tr>
+                                    <td></td>
+                                    <td>{$activity.description}
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <!--  END: description -->
+
+
+                {/foreach}
+
+                </tbody>
+                <!-- END: row -->
+            </table>
+
+        </div>
+
+        <div role="tabpanel" class="tab-pane" id="emails">
+            <table class="list view table-responsive subpanel-table">
+                <thead>
+                <tr class="footable-header">
+                    <th><img src="include/images/blank.gif" width="1" height="1" alt=""></th>
+                    <th>{$mod.LBL_LIST_SUBJECT}</th>
+                    <th>{$mod.LBL_LIST_STATUS}</th>
+                    <th>{$mod.LBL_LIST_CONTACT}</th>
+                    <th>{$mod.LBL_LIST_DATE}</th>
+                </tr>
+                </thead>
+                <tbody>
+                {foreach from=$emailsList key=k item=activity}
+
+                    <!-- BEGIN: row -->
+                    <tr>
+                        <td><img src={$activity.image}></td>
```

Comment:
It would be better to add a CSS class in the DOM here, So that we can control the appearance of the image, when we maintain or change the theme in the future.